mPowerSDK/mPowerSDK/TasksAndSteps/APHScoreCalculator.m

```diff
@@ -108,4 +115,61 @@ + (NSArray*)convertPostureOrGain:(NSURL *)url
     return  gaitItems;
 }
 
+- (double)scoreFromTappingResult:(ORKTappingIntervalResult *)result
```

Comment:
Was wondering about the scoring routines being defined in the app code rather than the mPower SDK. Was the scoring done differently between the app store and the open versions? If so, is there any reason you can think of why they *should* be different (I can't, offhand), or can we just roll the app store version of the scoring into the mPower SDK?